Sources/EdgeNetworkHandlers/EdgeHitProcessor.swift

```diff
@@ -224,4 +226,15 @@ class EdgeHitProcessor: HitProcessing {
         return requestHeaders
     }
 
+    private func getCustomPathFromEvent(_ event: Event) -> String? {
```

Comment:
nit: Rename to `getCustomRequestPath(from event: Event) -> String?`. Then call by `getCustomRequestPath(from: myEvent)`